### DISCUSSION TITLE
Is there a way to get the zoom property from another Visualizer in open3d

### DISCUSSION THREAD

**Discussion by MeyerBuaharon** [isl-org/Open3D#3889] [Q&A] [answered]
Is there a way to get my current zoom of the open3d.visualization.Visualizer because I want to get it and set it in Visualizer.

I wrote this to set up the zoom `o3d.visualization.ViewControl.set_zoom(self.vis.get_view_control(), 1.1)` but instead of the 1.1 I want to do a get of another vis zoom property.

**Comment by errissa** [comment, marked as answer]
With the new visualizer we don't have a concept of a "zoom" level like the old visualizer. However, you can retrieve the current FOV from the camera class then save it off and use it later to setup a new camera:
```
cam1 = o3dscene.camera
fov = cam1.get_field_of_view() # Save this somewhere

# In other visualizer - get the camera and set its FOV
cam2 = o3dscene.camera
cam2.set_projection(fov, aspect_ratio, far_plane, Camera.Projection.Perspective)
```
